<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_flashcard</class>
 <widget class="QMainWindow" name="window_flashcard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flashcard</string>
  </property>
  <widget class="QWidget" name="widget_central">
   <widget class="QListWidget" name="list_cluster">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_learn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>I knew this word</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_unlearn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>I didn't know this word</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_learned">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
